### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by MohamedTaha880**

### CODE REVIEW

File: src/modules/node/classes/Common/NodeFactory.class.php [hunk 1]
```diff
@@ -464,6 +465,25 @@ public static function requestAction(Environment $env, $action, array $form_data
             $node->setAuthor($form_data['author']);
           }
 
+          if(isset($form_data['password'])){
```
Comment (Aldus83) [inline src/modules/node/classes/Common/NodeFactory.class.php:468]: @MohamedTaha880 I remember the password check / password repeat etc. was already implemented somewhere else, can you have a look in code (i.e. grep password_rp) to make sure we don't add any redundancy?

Comment (MohamedTaha880) [reply src/modules/node/classes/Common/NodeFactory.class.php:468]: @Aldus83 
Screenshot 2024-04-25 130352
There is no implementation except in the user hook and only in the case of the register
And here in our case, it is not called

File: src/modules/shadow/assets/js/shadow.js [hunk 2]
```diff
@@ -175,7 +175,7 @@ function submitShadow() {
       } else {
         form_items[fieldName].push(fieldValue); // Push trimmed field value
       }
-    }
+    // }
```
Comment (Aldus83) [inline src/modules/shadow/assets/js/shadow.js:178]: fix commented bracket

**Review by Aldus83 — APPROVED** [review]
please fix the commented bracket

**Pull request merged**